### DISCUSSION TITLE
blog/aws-credentials-for-dotnet-applications/

### DISCUSSION THREAD

**Discussion by giscus** [codewithmukesh/discussions#20] [General]
# blog/aws-credentials-for-dotnet-applications/

In this article, we will go through the best practices for configuring & managing AWS Credentials for .NET applications. We will walk through various ways to secure the AWS Credentials by using app settings, environment variables, AWS CLI profiles, and IAM Roles.

https://codewithmukesh.com/blog/aws-credentials-for-dotnet-applications/

**Comment by myzani** [comment]
What a great article! Looking forward using public & private certificate in api gateway with lambda & .net core api. Keep rockin' Mukesh!
